control-plane/subcommand/server-acl-init/command.go

```diff
@@ -126,13 +126,13 @@ func (c *Command) init() {
 		"Toggle for creating a client agent token. Default is true.")
 
 	c.flags.BoolVar(&c.flagSyncCatalog, "sync-catalog", false,
-		"Toggle for creating a catalog sync policy.")
+		"Toggle for creating acl policies and rolebindings for sync catalog.")
```

Comment:
I think this might a bit better so we don't have to enumerate all the things. Otherwise, we have to mention, ACL policy, role, and binding rule explicitly.
```suggestion
		"Toggle for configuring ACL login for sync catalog.")
```